build/images/ovs/Dockerfile.ubi

```diff
@@ -53,7 +53,6 @@ RUN rm -f /etc/yum.repos.d/* && mv /tmp/CentOS.repo /etc/yum.repos.d/CentOS.repo
     # Disable the default redhat.repo. This substitutes `subscription-manager config --rhsm.manage_repos=0`
     # as subscription-manager is not supported running in containers.
     sed -i.bak "s/^manage_repos = .$/manage_repos = 0/g" /etc/rhsm/rhsm.conf && \
-    yum clean all -y && yum reinstall yum -y && \
```

Comment:
not sure what this was for in the first place. I'll add @ksamoray to reviewers.

Reply:
Not sure either as it's been a while. If we produce a functional image then I guess that we don't need it :)